openapi.yaml

```diff
@@ -296,7 +296,30 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
-  
+
+  /signers/tickers:
+    get:
+      summary: Get the signers known by the aggregator
+      description: |
+        Returns the signers party id and, if available, their pool ticker
+      responses:
+        "200":
+          description: Signers found
+          content:
+            application/json:
```

Comment:
This should also return the network name (for the explorer to know how to link back to the Cardano explorer)